source/UIAHandler/utils.py

```diff
@@ -362,3 +363,11 @@ def _shouldSelectivelyRegister() -> bool:
 		return False
 	else:
 		return winVersion.getWinVer() >= winVersion.WIN11_22H2
+
+
+def _shouldUseWtNotifications() -> bool:
+	"Determines whether to use notifications for new text reporting in Windows Terminal."
+	# Once we're ready to switch this on, update behaviorOfDefault in the
+	# config spec, then change the test below to check that the config value is
+	# not equal to DIFFING (assuming no other strategy is implemented in the meantime).

```

Comment:
I don't think this is required, `.value` should return `WtStrategyFlag.NOTIFICATIONS` if `default` is selected and `behaviorOfDefault='notifications'`

Reply:
Not according to my tests: `value` gets `DEFAULT` when default is selected, regardless of `behaviorOfDefault`.

Reply:
Sorry it should not be `.value`

Reply:
.calculated() or just the config item should resolve to the calculated value